src/Kiota.Builder/Lock/KiotaLockComparer.cs

```diff
@@ -28,6 +28,7 @@ public int GetHashCode([DisallowNull] KiotaLock obj)
             (string.IsNullOrEmpty(obj.DescriptionLocation) ? 0 : obj.DescriptionLocation.GetHashCode(StringComparison.OrdinalIgnoreCase)) * 41 +
             (string.IsNullOrEmpty(obj.DescriptionHash) ? 0 : obj.DescriptionHash.GetHashCode(StringComparison.OrdinalIgnoreCase)) * 37 +
             (string.IsNullOrEmpty(obj.ClientClassName) ? 0 : obj.ClientClassName.GetHashCode(StringComparison.OrdinalIgnoreCase)) * 31 +
+            (string.IsNullOrEmpty(obj.ClientClassAccessModifier) ? 0 : obj.ClientClassAccessModifier.GetHashCode(StringComparison.OrdinalIgnoreCase)) * 61 +
```

Comment:
Does it makes sense to move this to the top? For readability of the increasing primes?